magprime/templates/preregistration/attendee_donation_form.html

```diff
@@ -1,37 +0,0 @@
-{% extends "./preregistration/preregbase.html" %}
-{% block title %}Additional Donation{% endblock %}
-{% block backlink %}{% endblock %}
-{% block content %}
-<div class="masthead">
-</div>
-<div class="panel panel-default">
-{% if attendee.paid == c.NOT_PAID or attendee.overridden_price %}
-    <h2> Badge Payment for {{ attendee.full_name }} </h2>
-
-    You've registered for {{ c.EVENT_NAME }} at the price of ${{ attendee.overridden_price|default(attendee.badge_cost|round(2)) }}{% if attendee.amount_extra %} and you've also kicked in ${{ attendee.amount_extra|round(2) }}{% endif %}; your total outstanding balance is ${{ attendee.amount_unpaid|round(2) }}.
```

Comment:
This entire template was literally just so we could show them the overridden_price instead of their actual badge price. We don't need it anymore.